packages/reputation-miner/ReputationMiner.js

```diff
@@ -257,9 +257,7 @@ class ReputationMiner {
     // We update colonywide sums first (children, parents, skill)
     // Then the user-specifc sums in the order children, parents, skill.
 
-    // Converting to decimal, since its going to be converted to hex inside `insert`
-    const skillIdDecimal = new BN(skillId, 16).toString();
-    await this.insert(colonyAddress, skillIdDecimal, userAddress, score, updateNumber);
+    await this.insert(colonyAddress, skillId, userAddress, score, updateNumber);
```

Comment:
Do we need to `breakKeyIntoElements` only in the `insert` to compose it again?

Reply:
Heh, probably not. It's been like that for a while, but yeah, I can't see a reason.